src/ui/print.jl

```diff
@@ -107,12 +117,14 @@ function print_reports(io::IO,
 
     arg = :color => get(io, :color, false)
     with_bufferring(arg) do io
+        io = depth_lim_cxt(io)
```

Comment:
Given that `print_reports(::IO, res::[JETToplevelResult|JETCallResult])`, defined at the top of the file, serves as the entry point for report printing, it would be beneficial to manage all the `IOContext` settings there in one centralized location.

Reply:
By centralizing it in this way, it should also become more straightforward to make the feature configurable, by using an implementation like `get(res.jetconfigs, :stacktraces_type_limited, true)`.

Reply:
I'll resolve this, but maybe you can comment after my force-push, to suggest how to incorporate the `stacktraces_type_limited` setting